src/qt/locale/bitcoin_en.ts

```diff
@@ -4084,7 +4084,7 @@ for example: alertnotify=echo %%s | mail -s &quot;Bitcoin Alert&quot; admin@foo.
     </message>
     <message>
         <location line="+3"/>
-        <source>Copyright (C) 2009-%i The Bitcoin Core Developers</source>
+        <source>Copyright (C) 2009-%i The Bitcoin Core developers</source>
```

Comment:
You need to change this in ``LicenseInfo()`` in init.cpp and not here in any of these files.